packages/blade/src/components/Box/BaseBox/types/propsTypes.ts

```diff
@@ -142,6 +135,32 @@ type BaseBoxVisualProps = MakeObjectResponsive<
     borderTopRightRadius: keyof Border['radius'];
     borderBottomRightRadius: keyof Border['radius'];
     borderBottomLeftRadius: keyof Border['radius'];
+  } & PickCSSByPlatform<
+    | 'backgroundImage'
+    | 'backgroundSize'
+    | 'backgroundPosition'
+    | 'backgroundOrigin'
+    | 'backgroundRepeat'
+  >
+>;
+
+// Visual props that are specific BaseBox
+// This is used to ensure some of the more flexible BaseBox props are not passed to Box
+type BaseBoxVisualProps = MakeObjectResponsive<
+  {
+    backgroundColor:
+      | BackgroundColorString<'feedback'>
+      | BackgroundColorString<'surface'>
+      | BackgroundColorString<'action'>
+      | (string & Record<never, never>);
+    backgroundImage: CSSObject['backgroundImage'];
+    backgroundSize: CSSObject['backgroundSize'];
+    backgroundPosition: CSSObject['backgroundPosition'];
+    backgroundOrigin: CSSObject['backgroundOrigin'];
+    backgroundRepeat: CSSObject['backgroundRepeat'];
```

Comment:
I can see these are added to `CommonBoxVisualProps` now so we can remove from here right?

Reply:
Yeah forgot to remove those